### PR TITLE
SREP-1143: Add pagerduty-operator

### PR DESCRIPTION
**Pull request by lisa**
Adds openshift/pagerduty-operator (which is still in development).

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/pagerduty-operator/openshift-pagerduty-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+base_images:
+ base:
+   name: '4.0'
+   namespace: ocp
+   tag: base
+canonical_go_repository: github.com/openshift/pagerduty-operator
+images:
+- dockerfile_path: build/Dockerfile
+  from: base
+  to: <operator name>
```
Comment (droslean) [inline ci-operator/config/openshift/pagerduty-operator/openshift-pagerduty-operator-master.yaml]: ?

Comment (lisa) [reply ci-operator/config/openshift/pagerduty-operator/openshift-pagerduty-operator-master.yaml]: Fixed.

**Pull request merged**